app/src/main/java/com/bogdan/codeforceswatcher/features/actions/redux/requests/ActionsRequests.kt

```diff
@@ -23,19 +23,10 @@ class ActionsRequests {
     ) : Request() {
 
         override suspend fun execute() {
-            for (i in 0..2) {
```

Comment:
Well, it's a dirty hack. Your actual problem comes from `getUsers` requests, which are executed in parallel and fails (as it was before). Adding delay for `isInitiated == false` in `FetchActions` fixes the problem. But try to figure out better solution 😉(For example, you can fetch actions when users are successfully fetched, but check the case when there are no users).